server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -1022,10 +1019,10 @@ ClusterState readClusterStateInParallel(
 
         LatchedActionListener<IndexRoutingTable> routingTableLatchedActionListener = new LatchedActionListener<>(
             ActionListener.wrap(response -> {
-                logger.debug("Successfully read cluster state component from remote");
+                logger.debug(() -> new ParameterizedMessage("Successfully read index-routing for index {}", response.getIndex().getName()));
                 readIndexRoutingTableResults.add(response);
             }, ex -> {
-                logger.error("Failed to read cluster state from remote", ex);
+                logger.error(() -> new ParameterizedMessage("Failed to read index-routing from remote"), ex);
```

Comment:
For each of the logger messages can we add more details around which entity failed

Reply:
We add in while reading writing through the CheckSumWritableBloblStoreFormat here: https://github.com/opensearch-project/OpenSearch/blob/7f0ff145ee3a9725833512967459f0a62a591d2b/server/src/main/java/org/opensearch/repositories/blobstore/ChecksumWritableBlobStoreFormat.java#L88